src/models/emailServiceInt/templates/invitationAdmin.tsx

```diff
@@ -1,23 +1,41 @@
 import Html from '@kitajs/html'
 import { SendMailOptions } from 'nodemailer'
+
 import { Env } from '../../../env/index.js'
+import CompanyHouseEntity from '../../companyHouseEntity.js'
 
 export default {
   name: 'connection_invite_admin' as const,
-  template: async function (env: Env, params: { pin: string; address: string }): Promise<SendMailOptions> {
+  template: async function (
+    env: Env,
+    params: { pin: string; receiver: string; address: string }
+  ): Promise<SendMailOptions> {
+    const companyHouse = new CompanyHouseEntity(env)
+    const localCompany = await companyHouse.localCompanyHouseProfile()
     return {
       to: env.get('EMAIL_ADMIN_ADDRESS'),
       from: env.get('EMAIL_FROM_ADDRESS'),
-      subject: 'Action required: process veritable invitation',
+      subject: `Postal Code for Verification: Invitation from ${params.receiver} on Veritable`,
       text: `
-        Action required: process veritable invitation    
-        PIN: ${params.pin},
-        Address: ${params.address},
+        Hi ${localCompany.company_name} Admin,
+        Please post this verification code to ${params.receiver}, to complete the verification process.
+
+        Address: ${params.address}
+
+        Verification Code: ${params.pin},
       `,
       html: await (
         <>
-          <h1>Action required: process veritable invitation</h1>
-          <p>Pin:</p>
+          <h1>{Html.escapeHtml(`Hi ${localCompany.company_name}`)}</h1>
+          <br />
+          <p>
+            {Html.escapeHtml(
+              `Please post this verification code to ${params.receiver}, to complete the verification process.`
+            )}
+          </p>
+
+          <br />
+          <p>Verification Code:</p>
```

Comment:
The copy between the subject and body is logically mixed up. When I do an invitation from alice to bob I get

```
Subject: Postal Code for Verification: Invitation from OFFSHORE RENEWABLE ENERGY CATAPULT on Veritable

Hi DIGITAL CATAPULT Admin,
Please post this verification code to OFFSHORE RENEWABLE ENERGY CATAPULT, to complete the verification process.

Address: OFFSHORE RENEWABLE ENERGY CATAPULT, Offshore House, Albert Street, Blyth, NE24 1LZ, Northumberland

Verification Code: 856478
```

In the `Subject` this should either be:

1. `Postal Code for Verification: Invitation from DIGITAL CATAPULT on Veritable`
2. `Postal Code for Verification: Invitation to OFFSHORE RENEWABLE ENERGY CATAPULT on Veritable`

in order to logically make sense. I think you need to ask Esther which is preferable.

Reply:
oops, yea, currently used receiver rather than local, have been updated.